test/TMxpStubClient.h

```diff
@@ -133,7 +135,12 @@ class TMxpStubClient : public TMxpClient {
 
     }
 
-    void publishEntity(const QString& name, const QString& value) override {}
+    void publishEntity(const QString& name, const QString& value) override
+    {
+        qDebug() << qsl("publishEntity([%1], [%2])").arg(name, value);
```

Comment:
I like to see the class name in such debug output - it helps to narrow down where the message is coming from:
```suggestion
        qDebug().noquote() << qsl("TMxpStubClient::publishEntity([%1], [%2]) INFO - called").arg(name, value);
```

the `noquote()` stops Qt presenting the whole lot wrapped in `"`...`"`. Also how about the same sort of thing for the `setLink(...)` method before here?